wpilib/wpilib/analoggyro.py

```diff
@@ -64,18 +64,13 @@ def __init__(self, channel, center=None, offset=None):
             
         self.analog = channel
 
-        self.voltsPerDegreePerSecond = AnalogGyro.kDefaultVoltsPerDegreePerSecond
-        self.analog.setAverageBits(AnalogGyro.kAverageBits)
-        self.analog.setOversampleBits(AnalogGyro.kOversampleBits)
-        sampleRate = AnalogGyro.kSamplesPerSecond \
-                * (1 << (AnalogGyro.kAverageBits + AnalogGyro.kOversampleBits))
-        AnalogInput.setGlobalSampleRate(sampleRate)
-        if not hal.HALIsSimulation():
-            Timer.delay(1.0)
-        
+        if self._gyroHandle == 0:
+            self._gyroHandle = hal.initializeAnalogGyro(self.analog.port)
```

Comment:
I've thought about using that convention, but I think the rest of the code doesn't do that, so let's not start now.